### PR TITLE
(SIMP-2226) Create chkrootkit module

### PR DESCRIPTION
**Pull request by jeefberkey**
This material used to be in simplib.

SIMP-2226 #close
SIMP-1679 #comment created chkrootkit module

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,85 @@
...
+3. [Usage - Configuration options and additional functionality](#usage)
+4. [Reference - An under-the-hood peek at what the module is doing and how](#reference)
+5. [Limitations - OS compatibility, etc.](#limitations)
+6. [Development - Guide for contributing to the module](#development)
+    * [Acceptance Tests - Beaker env variables](#acceptance-tests)
+
+
+## Description
+
+chkrootkit is a Puppet module that manages settings regarding users and user creation.
```
Comment (trevor-vaughan) [inline README.md]: Incorrect

File: manifests/init.pp [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    minute   => $minute,
+    hour     => $hour,
+    month    => $month,
+    monthday => $monthday,
+    weekday  => $weekday,
+    require  => Package['chkrootkit']
+  }
+
+  package { 'chkrootkit': ensure => 'latest' }
+}
```
Comment (trevor-vaughan) [inline manifests/init.pp]: Fix

File: metadata.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
+{
+  "name":    "simp-chkrootkit",
+  "version": "0.1.0",
```
Comment (trevor-vaughan) [inline metadata.json]: Start at 0.0.1

File: spec/acceptance/nodesets/default.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+      - default
+      - master
+    platform:   el-7-x86_64
+    box:        centos/7
+    hypervisor: vagrant
+  client:
+    roles:
+      - client
+      - agent
+    platform:   el-7-x86_64
```
Comment (trevor-vaughan) [inline spec/acceptance/nodesets/default.yml]: Need an EL6 box in here somewhere

File: manifests/init.pp (hunk 2 above)
Comment (jeefberkey) [inline manifests/init.pp]: What's wrong with this?

Comment (trevor-vaughan) [inline manifests/init.pp]: @jeefberkey No newline at the end of the file. It's not showing in this view for some reason.

**Pull request merged**